src/experiments/preferences/api.js

```diff
@@ -13,36 +13,17 @@ Cu2.import("resource://gre/modules/ExtensionPreferencesManager.jsm");
 
 const TRR_URI_PREF = "network.trr.uri";
 const TRR_DISABLE_ECS_PREF = "network.trr.disable-ECS";
-const TRR_MODE_PREF = "network.trr.mode";
 
 ExtensionPreferencesManager.addSetting("dohRollout.state", {
   prefNames: [
     TRR_URI_PREF,
     TRR_DISABLE_ECS_PREF,
```

Comment:
To be clear, we're ok with any potential reset of these prefs via EPM?  It seems that is highly unlikely in the releases we'll roll out to, but it is the design of EPM to do so.

Since this is done, I will request a change in the schema.

Reply:
Let's limit the scope of this PR to 69/70/71 at this point. For these releases, the only possibilities of these prefs being changed are manually by users, or secure proxy. Let's make sure we have the correct logic to back off when we detect these changes.

Reply:
@mixedpuppy I left these prefs as is to minimize code change. As I've already moved the `network.trr.mode` away from this pattern/tool, I'm comfortable moving others. Logging this comment as a new issue to revise in the next release.

Reply:
@mixedpuppy Scratch that — you already opened one! Thank you. #168